client/my-sites/customer-home/main.jsx

```diff
@@ -93,6 +95,7 @@ const Home = ( {
 					<div className="customer-home__layout-col customer-home__layout-col-right">
 						<Secondary />
 					</div>
+					{ isWithinBreakpoint( '<1040px' ) && <ListEnd /> }
```

Comment:
@arunsathiya Any of the checks from @automattic/viewport only work on page-load. So basically if we resize the browser we'll see behavior of whatever viewport size we started with.

Reply:
By the way, is it better to take this approach or just use CSS to hide it? Calypso tends to do the latter, but I think the Customer Home uses ` isWithinBreakpoint ` more. :)

Reply:
The tradeoff is rendering more than we need (heavier page load) vs not being as responsive (folks who like resizing windows). So it depends on the page. 

Components themselves should be somewhat responsive.

One alternative might be to debounce how often the viewport calculations run (to get more accurate measurements), instead of only doing this once.